delta_backend/src/ConversionLayout.py

```diff
@@ -140,15 +140,15 @@
       "expression": {
         "expressionName": "Date Convert",
         "expressionType": "DATECONVERT",
-        "expressionRule": "format:%Y%m%d"
+        "expressionRule": "%Y%m%d"
       }
     },
     {
       "fieldNameFHIR": "primarySource",
       "fieldNameFlat": "PRIMARY_SOURCE",
       "expression": {
         "expressionName": "Not Empty",
-        "expressionType": "NOTEMPTY",
+        "expressionType": "BOOLEAN",

```

Comment:
A few `expressionName` and `expressionType` mismatches in this file.

Maybe a question for later, but why do we even have both? I'm assuming only the `expressionType` matters?